riscv/decode_macros.h

```diff
@@ -171,7 +171,6 @@ static inline bool is_aligned(const unsigned val, const unsigned pos)
 #define require_vector(alu) \
   do { \
     require_vector_vs; \
-    require_extension('V'); \
```

Comment:
Same question as https://github.com/riscv-software-src/riscv-isa-sim/pull/1701/commits/adfb8e4b43beace123208841a722048ea875876d#r1648223166

...but I guess the mechanism might be different here?